bundles/org.openhab.binding.visualcrossing/README.md

```diff
@@ -0,0 +1,359 @@
+| uv-index        | Number               | R          | A value between 0 and 10 indicating the level of ultra violet (UV) exposure for that hour or day. 10 represents high level of exposure, and 0 represents no exposure. The UV index is calculated based on amount of short wave solar radiation which in turn is a level the cloudiness, type of cloud, time of day, time of year and location altitude. Daily values represent the maximum value of the hourly values.                  |
+| sunrise         | String               | R          |                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| sunrise-epoch   |                      | R          |                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| sunset          | String               | R          |                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| sunset-epoch    |                      | R          |                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| moon-phase      | Number               | R          | Represents the fractional portion through the current moon lunation cycle ranging from 0 (the new moon) to 0.5 (the full moon) and back to 1 (the next new moon). See [How to include sunrise, sunset, moon phase, moonrise and moonset data into your API requests](https://www.visualcrossing.com/resources/documentation/weather-api/how-to-include-sunrise-sunset-and-moon-phase-data-into-your-api-requests/)                      |
```

Comment:
Please remove these copy/paste text parts from the description as they are not from the binding perspective, but from the API perspective, that should be transparant to the user. (please fix others too)
```suggestion
| moon-phase      | Number               | R          | Represents the fractional portion through the current moon lunation cycle ranging from 0 (the new moon) to 0.5 (the full moon) and back to 1 (the next new moon).                       |
```